src/main/java/org/mskcc/cbio/portal/util/ConsoleUtil.java

```diff
@@ -138,8 +138,10 @@ public static OptionSet parseStandardDataAndMetaOptions(String[] args, String de
 			parser.accepts( "loadMode", "direct (per record) or bulk load of data" )
 			          .withRequiredArg().describedAs( "[directLoad|bulkLoad (default)]" ).ofType( String.class );
 		}
+		parser.accepts("overwrite-existing",
+				"Enables re-uploading molecular data that already exist for the given profile and sample.").withOptionalArg().describedAs("overwrite-existing").ofType(String.class);
```

Comment:
```suggestion
				"Enables overwriting data if it turns out it already exists in DB").withOptionalArg().describedAs("overwrite-existing").ofType(String.class);
```
not all data is sample based, or molecular, as shown in this clinical event example.

Reply:
see here https://github.com/cBioPortal/cbioportal-core/pull/47/commits/16f62955e5e4c1e26017d8290fac6c299dd90e19